implementations/rust/ockam/ockam_identity/src/credential_issuer.rs

```diff
@@ -1,256 +0,0 @@
-use crate::alloc::string::ToString;
```

Comment:
It's now an empty file

Reply:
There are others, could you please check for empty files?

Reply:
That looks like the result of merge conflicts resolution. Done.